components/Officers.js

```diff
@@ -1,7 +1,143 @@
+		image: "samarth.png",
+		email: "ssrin027@ucr.edu",
+		linkedin: "www.linkedin.com/in/samarth-srinivasa",
+		github: "https://github.com/SamarthSrinivasa",
+		bio: "I am a third year data science major at UCR! Contact me if you have any questions about talking to industry leaders and companies in the field of computer science, or any other questions you might have about ACM! Other than School and ACM, I enjoy playing music, chess, and cooking.",
+	},
+	{
+		name: "Ashley Kim",
+		pronoun: "She/Her",
+		position: "Co-Marketing Chair",
+		image: "ashley.jpg",
+		email: "akim291@ucr.edu",
+		linkedin: "www.linkedin.com/in/ashley-kim-838893196",
+		github: "",
+		bio: `Hi everyone! I'm Ashley and I'm a 3rd year CS major. My hobbies outside of class include digital illustration, making fashion pinterest boards, and baking cookies for my friends.`,
+	},
+	{
+		name: "Ashwin Jagadish",
+		pronoun: "He/Him",
+		position: "Marketing Co-Chair",
+		image: "ashwin.jpg",
+		email: "ajaga008@ucr.edu",
+		linkedin: "www.linkedin.com/in/ashwin-jagadish",
+		github: "",
+		bio: `Hi, my name is Ashwin, and I'm a 3rd year Computer Science with Business Applications major. I'm looking forward to being ACM's Marketing Chair this year and creating more engagement within the club.`,
+	},
+	{
+		name: "Johnnie Dinh",
+		pronoun: "He/Him",
+		position: "Secretary",
+		image: "johnnie.png",
+		email: "kdinh032@ucr.edu",
+		linkedin: "https://www.linkedin.com/in/johnniedinh/",
+		github: "https://github.com/johnniedinhCS",
+		bio: "Friendliest secretary around (and maybe the coolest); Resident keyboard warrior of ACM. Always happy to help, so feel free to shoot me a message, even at 3 AM.",
+	},
+	{
+		name: "Sachin Chopra",
+		pronoun: "He/Him",
+		position: "Alumni Chair",
+		image: "sachin.png",
+		email: "schop021@ucr.edu",
+		linkedin: "https://www.linkedin.com/in/sachinkchopra",
+		github: "https://github.com/SafeDuck",
+		bio: `Hey ACM folks, I'm Sachin! I study CS and love to discuss the theories of the subject. You can catch me doing some origami, playing guitar, or asking professors way too many questions. I hope you find your community and enjoy your time here.`,
+	},
+	{
+		name: "Divyank Shah",
+		pronoun: "He/Him",
+		position: "Professional Development Chair",
+		image: "divyank.jpg",
+		email: "dshah048@ucr.edu",
+		linkedin: "https://www.linkedin.com/in/divyank-shah/",
+		github: "https://github.com/shahdivyank",
+		bio: `Hey! I'm Divyank and I am a 1st year CS major! I am your professional development chair this year and am excited to create new opporuntites for everyone! Outside of school I love to talk tech, go on hikes, or take a bike ride!`,
+	},
+];
 
 const Officers = () => {
-	return <div>Officers</div>;
+	return (
+		<div className='container flex flex-col items-center justify-center p-4 mx-auto'>
+			<h1 className='font-lexend text-3xl font-medium text-center'>
+				Meet our Board!
+			</h1>
+
+			<div className='profileGrid grid grid-cols-2'>
```

Comment:
Can you use bootstrap grid instead of the inbuilt cols? Something like [this!](https://github.com/acm-ucr/acm-dev/blob/dev/components/Upcoming.js#L24)